test/JDBC/upgrade/13_6/preparation/sys-configurations-vu-prepare.sql

```diff
@@ -0,0 +1,47 @@
+CREATE DATABASE db_sys_configurations;
```

Comment:
Ideally, we should create test objects with test name and that way no need to use test specific database